src/app/views/censusdata/create.html

```diff
@@ -72,7 +51,7 @@
                 <label class="control-label col-md-3">Weather:</label>
                 <div class="col-md-9">
                 <div class="checkbox">
-                    <label ng-repeat="weather in data.weatherInfo"><input type="radio" name="weatherInfo" ng-model="censusForm.weatherInfo" value="{{weather.Id}}"> {{ weather.Description }} &nbsp;</label>      
+                    <label ng-repeat="weather in data.weatherInfo"><input type="radio" name="weatherInfo" ng-model="censusForm.weatherInfoId" value="{{weather.Id}}"> {{ weather.Description }} &nbsp;</label>      
```

Comment:
Oh, sorry, I missed that. I saw line 74/53 with `class="checkbox"`at first, but now I see this is `<input type="radio"`. Great!